From eec16585b22aa79251b2e97c8a4ed39110e7d94c Mon Sep 17 00:00:00 2001
From: Quintin Hill <Quintin.Hill@arqiva.com>
Date: Fri, 12 Jan 2018 11:15:34 +0000
Subject: [PATCH 1/3] condor_includes: add includes to condor_fix_assert.h

This ensures that when compiling on Windows that condor_debug.h has all required macros defined.
---
 src/condor_includes/condor_fix_assert.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/condor_includes/condor_fix_assert.h b/src/condor_includes/condor_fix_assert.h
index 15072998f4..8ae50b9260 100644
--- a/src/condor_includes/condor_fix_assert.h
+++ b/src/condor_includes/condor_fix_assert.h
@@ -23,6 +23,11 @@
 #ifndef WIN32	/* on Win32, we do EXCEPT instead of assert */
 #include <assert.h>
 #else
+#if HAVE_CONFIG_H
+#include "config.h"
+#endif
+#include "condor_header_features.h"
+#include "condor_system.h"
 #include "condor_debug.h"
 #endif	/* of else ifndef WIN32 */
 
-- 
2.14.3

